doc/releases/changelog-0.21.0.md

```diff
@@ -4,22 +4,131 @@
-  [(#2103)](https://github.com/PennyLaneAI/pennylane/pull/2103)
+<h4>Tapering off qubits</h4>
+
+* Functions for tapering qubits based on molecular symmetries have been added.
+  [(#1966)](https://github.com/PennyLaneAI/pennylane/pull/1966)
+  [(#1974)](https://github.com/PennyLaneAI/pennylane/pull/1974)
+  [(#2041)](https://github.com/PennyLaneAI/pennylane/pull/2041)
+  [(#2042)](https://github.com/PennyLaneAI/pennylane/pull/2042)
+
+  With this functionality, a molecular Hamiltonian and the corresponding Hartree-Fock (HF) state can be transformed to a new Hamiltonian and HF state that acts on a reduced number of qubits, respectively.
 
-  This function combines redundant terms in a Hamiltonian and eliminates terms with a coefficient
-  smaller than a cutoff value. The new function makes construction of molecular Hamiltonians more
-  efficient. For LiH, as an example, the time to construct the Hamiltonian is reduced roughly by a
-  factor of 20.
+  ```python
+  from pennylane import hf
+  from pennylane import numpy as np
+
+  symbols = ["He", "H"]
+  geometry = np.array([[0.0, 0.0, 0.0], [0.0, 0.0, 1.4588684632]])
+  mol = hf.Molecule(symbols, geometry, charge=1)
+  H = hf.generate_hamiltonian(mol)(geometry)
+  n_qubits, n_elec = len(H.wires), mol.n_electrons
 
-* The JAX interface now supports evaluating vector-valued QNodes. Vector-valued
-  QNodes include those with:
+  generators, paulix_ops = hf.generate_symmetries(H, n_qubits)
+  paulix_sector = [1, -1, -1]
+  opt_sector = hf.optimal_sector(H, generators, n_elec)
+  H_tapered = hf.transform_hamiltonian(H, generators, paulix_ops, opt_sector)
+  hf_tapered = hf.transform_hf(generators, paulix_ops, paulix_sector,
+                                    n_elec, n_qubits)
+  ```
+  ```pycon
+  >>> print(H_tapered)
+    ((-1.7997297644914574+0j)) [I0]
+  + ((-0.10492941956079854+0j)) [X0]
+  + ((0.10492941956079856+0j)) [X1]
+  + ((0.5675134088336165+0j)) [Z1]
+  + ((0.5675134088336168+0j)) [Z0]
+  + ((-0.14563730440190722+0j)) [Y0 Y1]
+  + ((-0.10492941933657857+0j)) [X0 Z1]
+  + ((0.09337410512815508+0j)) [Z0 Z1]
+  + ((0.10492941933657857+0j)) [Z0 X1]
+  >>> hf_tapered
+  tensor([1, 1], requires_grad=True)
+  ```
+
+<h4>New tensor network templates</h4>
+
+* A tensor network templates module has been added. Quantum circuits with the shape
+  of a matrix product state tensor network can now be easily implemented.
+  Motivation and theory can be found in [arXiv:1803.11537](https://arxiv.org/abs/1803.11537).
+  [(#1871)](https://github.com/PennyLaneAI/pennylane/pull/1871)
+
+  An example circuit that uses the `MPS` template is:
+  ```python
+  import pennylane as qml
+  import numpy as np
+
+  def block(weights, wires):
+      qml.CNOT(wires=[wires[0],wires[1]])
+      qml.RY(weights[0], wires=wires[0])
+      qml.RY(weights[1], wires=wires[1])
+
+  n_wires = 4
+  n_block_wires = 2
+  n_params_block = 2
+  template_weights = [[0.1,-0.3],[0.4,0.2],[-0.15,0.5]]
```

Comment:
@josh146 how come we'd like to make these inputs trainable? Is it because they are "weights"?

Reply:
yep! I worry someone will copy the example, and maybe then try to optimize it. With our latest `requires_grad` change, I'm not 100% sure this will work?

Reply:
Yep, makes sense. I've changed it.